### ISSUE TITLE
Square RequestCardNonce not defined error

### ISSUE THREAD

**Issue by kathmohr5** [square/connect-php-sdk#87]
I've been struggling with getting the Square API setup. I had it working for two years but a few weeks ago it stopped working. I reset the token and the shopping cart worked for 3 transactions then quit again. I've tried various potential solutions I've found online with no fix.  I continue to get this error.  Would someone have time to look at this with me?

<img width="349" alt="Screen Shot 2019-05-13 at 7 59 00 PM" src="https://user-images.githubusercontent.com/50601822/57661605-9f171380-75b9-11e9-8cf3-8ea4b75ecde7.png">

**Comment by StephenJosey** [comment]
Hey @kathmohr5 thanks for reaching out!

Based on that error it sounds like you're attempting to call `Charge`, but you aren't providing a `nonce`, which is what the payment form creates. Do you have a code snippet of where you're trying to call the `Charge` endpoint? I also noticed you mentioned "shopping cart" - is this something you've built, or is this a 3rd party integration?

**Comment by kathmohr5** [comment]
Hello. Thank you for responding.  I used the sample sqpaymentform files to create a payment system for a community parks and recreation department that integrates with the database I created for them a few years ago. I continued to work on the solution last night and am now seeing the nonce when I use Firefox inspector. However, the charge is not going into our square account and not getting a payment successful on the process-card.php page.  The page is blank.  I think I made progress but am still needing help.  Any ideas?

**Comment by StephenJosey** [comment]
Well it's blank depending on what you have in `process-card.php`. It will only show things if you've added something to display. Are you able to debug where you call `Charge` to see if the nonce is reaching that point? It sounds like your backend isn't successfully retrieving the nonce from your frontend.

**Comment by kathmohr5** [comment]
Thank you, Stephen.  I went to my file archive and brought back a version of the process-card.php file I had used before and it worked.  

Kathleen Mohr
Director of Technology/Professional Development
North Central Ohio Educational Service Center
Tiffin Center
928 West Market Street
Tiffin, Ohio  44883
Tiffin Office: 419-447-2929 x172
Cell: 419-939-3645
kmohr@ncoesc.org
Twitter: @kmohr

"Great things in business are never done by one person. They're done by a team of people."  - Steve Jobs
